libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/Encryptor.swift

```diff
@@ -46,11 +47,26 @@ public class Encryptor: NSObject {
     ///   - data: Data to encrypt
     ///   - key: Data representation of symmetric key to encrypt with
     /// - Returns: Encrypted data
-    @objc @available(swift, obsoleted: 1.0) // Objective-c only wrapper
+    @objc @available(*, deprecated, renamed: "encrypt(data:key:)") @available(swift, obsoleted: 1.0) // Objective-c only wrapper
     public static func encrypt(data: Data, using key: Data) throws -> Data {
         let symmetricKey = SymmetricKey(data: key)
         return try encrypt(data: data, using: symmetricKey)
     }
+    
+    /// Encrypts data with a given key
+    ///
+    /// - Parameters:
+    ///   - data: Data to encrypt
+    ///   - key: Data representation of symmetric key to encrypt with
+    /// - Returns: Encrypted data
+    @objc(encryptData:key:error:) @available(swift, obsoleted: 1.0) // Objective-c only wrapper
```

Comment:
Gave this a new name because the generated Objective-C `encryptWithData` felt confusing and made the key nullable so there wouldn't have to be a check on the key before calling it every time